src/vis_cpu/conversions.py

```diff
@@ -50,7 +50,7 @@ def enu_to_az_za(enu_e, enu_n, orientation="astropy", periodic_azimuth=True):
     ], "orientation must be either 'astropy' or 'uvbeam'"
 
     lsqr = enu_n ** 2.0 + enu_e ** 2.0
-    zeta = np.where(lsqr < 1.0, np.sqrt(1.0 - lsqr), 0.0)
+    zeta = np.where(lsqr < 1.0, np.sqrt(np.abs(1.0 - lsqr)), 0.0)
```

Comment:
Hmmm, for speed I guess just creating a mask array is going to be more efficient (and avoid the spurious warnings), i.e.

```
mask = lsqr < 1
zeta = np.zeros_like(lsqr)
zeta[mask] = np.sqrt(1 - lsqr[mask])
```
It's a few more lines, but avoids the extra calculations where the mask isn't true.

Reply:
Thanks, I've made this change.